tests/acceptance/features/webUISharingNotifications/shareWithGroups.feature

```diff
@@ -18,17 +18,17 @@ Feature: Sharing files and folders with internal groups
     And user "user2" has been added to group "grp1"
     And user "user2" has logged in using the webUI
 
-  @skip @yetToImplement
   Scenario: notifications about new share is displayed
-    Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been disabled
-    When user "user3" has shared folder "/simple-folder" with group "grp1"
-    And user "user3" has shared folder "/data.zip" with group "grp1"
-    Then the user should see 2 notifications on the webUI with these details
+    Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
+    When user "user3" shares folder "/simple-folder" with group "grp1"
+    And user "user3" shares folder "/data.zip" with group "grp1"
+    Then the user should see the notification bell on the webUI after a page reload
+    And the user should see notifications on the webUI with these details
       | title                                        |
       | "User Three" shared "simple-folder" with you |
       | "User Three" shared "data.zip" with you      |
     When the user re-logs in as "user1" using the webUI
-    Then the user should see 2 notifications on the webUI with these details
+    Then the user should see notifications on the webUI with these details
```

Comment:
I think use using the number of notifications for extra assertion is a good idea.